### PR TITLE
introduce calculation of missing value patterns

### PR DESCRIPTION
**Pull request by SultanOrazbayev**
This PR introduces a function to calculate missing patterns as described in https://github.com/ResidentMario/missingno/issues/101.

### CODE REVIEW

**Comment by vankesteren** [comment]
NB: I have left some comments / suggestions to this PR in [this comment](https://github.com/ResidentMario/missingno/issues/101#issuecomment-652896288) on issue #101

File: missingno/missingno.py [hunk 1]
```diff
@@ -545,3 +545,27 @@ def geoplot(df,
         plt.show()
     else:
         return ax
+
+
+def pattern(df, counter_name='mvcount', sort=True):
+    """
+    This function will replace the non-missing values with 1 and missing values with 0. The frequency of each pattern will be calculated in a column with the default name 'mvcount'. By default, the resulting dataframe will be sorted in order of descending frequency.
```
Comment (armando-fandango) [inline missingno/missingno.py:552]: What is the advantage of this function? Any specific use cases ?

Comment (SultanOrazbayev) [reply missingno/missingno.py:552]: I've used it in the past to understand the relative prevalence of specific patterns in missing-ness of data. It's useful for exploratory data analysis.

**Comment by ResidentMario** [comment]
This function may indeed be useful, but unfortunately I don't think `missingno` is an appropriate home for it. I'm already mulling over removing the existing filter utility functions.

**Pull request closed without merging**